### ISSUE TITLE
Deno panics on nextjs + next-translate app

### ISSUE THREAD

**Issue by MonstraG** [denoland/deno#26124]
Output of `deno run dev` with `RUST_BACKTRACE="full"`:
```
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: windows x86_64
Version: 2.0.0
Args: ["C:\\Users\\monst\\.deno\\bin\\deno.exe", "run", "--ext=js", "-A", "C:\\Users\\monst\\Downloads\\next-translate-deno-crash\\node_modules\\.bin\\../next/dist/bin/next", "dev"]

thread 'main' panicked at C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\deno_core-0.311.0\modules\map.rs:1201:5:
assertion `left == right` failed: Module already evaluated. Perhaps you've re-provided a module or extension that was already included in the snapshot? file:///C:/Users/monst/Downloads/next-translate-deno-crash/i18n.mjs (306)
  left: Evaluated
 right: Instantiated
stack backtrace:
   0:     0x7ff6462fa20d - node_api_get_module_file_name
   1:     0x7ff644adbee9 - node_api_get_module_file_name
   2:     0x7ff6462d7321 - node_api_get_module_file_name
   3:     0x7ff6462fe0c6 - node_api_get_module_file_name
   4:     0x7ff6462fd98a - node_api_get_module_file_name
   5:     0x7ff6462fd541 - node_api_get_module_file_name
   6:     0x7ff644939a13 - uv_close
   7:     0x7ff6462fe467 - node_api_get_module_file_name
   8:     0x7ff6462fe307 - node_api_get_module_file_name
   9:     0x7ff6462fe24f - node_api_get_module_file_name
  10:     0x7ff6462fe238 - node_api_get_module_file_name
  11:     0x7ff647e36dc4 - tree_sitter_javascript
  12:     0x7ff647e370f5 - tree_sitter_javascript
  13:     0x7ff647e3f8c5 - tree_sitter_javascript
  14:     0x7ff644c9cd6e - node_api_get_module_file_name
  15:     0x7ff644d01370 - node_api_get_module_file_name
  16:     0x7ff647a3f9c6 - CrashForExceptionInNonABICompliantCodeRange
  17:     0x7ff647a3db9e - CrashForExceptionInNonABICompliantCodeRange
  18:     0x7ff647a3db9e - CrashForExceptionInNonABICompliantCodeRange
  19:     0x7ff5c7c45f37 - <unknown>
  20:     0x7ff5c7c884a4 - <unknown>
  21:     0x7ff5c7c844dc - <unknown>
  22:     0x7ff5c7c8999b - <unknown>
  23:     0x7ff5c7c84695 - <unknown>
  24:     0x7ff647a3db9e - CrashForExceptionInNonABICompliantCodeRange
  25:     0x7ff647a3db9e - CrashForExceptionInNonABICompliantCodeRange
  26:     0x7ff647a81b8d - CrashForExceptionInNonABICompliantCodeRange
  27:     0x7ff647a3b71c - CrashForExceptionInNonABICompliantCodeRange
  28:     0x7ff647a3b26f - CrashForExceptionInNonABICompliantCodeRange
  29:     0x7ff646809a28 - node_api_get_module_file_name
  30:     0x7ff64680a605 - node_api_get_module_file_name
  31:     0x7ff64680a763 - node_api_get_module_file_name
  32:     0x7ff646811d81 - node_api_get_module_file_name
  33:     0x7ff6468114da - node_api_get_module_file_name
  34:     0x7ff646810f4f - node_api_get_module_file_name
  35:     0x7ff64680b18c - node_api_get_module_file_name
  36:     0x7ff646692e9d - node_api_get_module_file_name
  37:     0x7ff64667ef80 - node_api_get_module_file_name
  38:     0x7ff644d48627 - node_api_get_module_file_name
  39:     0x7ff644d46daf - node_api_get_module_file_name
  40:     0x7ff64458b898 - uv_mutex_destroy
  41:     0x7ff6441ecb28 - uv_mutex_destroy
  42:     0x7ff6449050d2 - uv_close
  43:     0x7ff6448a867c - uv_close
  44:     0x7ff644923aa4 - uv_close
  45:     0x7ff6442e5e0d - uv_mutex_destroy
  46:     0x7ff644a067b5 - node_api_get_module_file_name
  47:     0x7ff6444398df - uv_mutex_destroy
  48:     0x7ff64493c07c - uv_close
  49:     0x7ff6442b221f - uv_mutex_destroy
  50:     0x7ff644a06896 - node_api_get_module_file_name
  51:     0x7ff647dd8c8c - tree_sitter_javascript
  52:     0x7ffaef8d257d - BaseThreadInitThunk
  53:     0x7ffaf078af28 - RtlUserThreadStart
  ```
  
Reproduction repository: https://github.com/MonstraG/next-translate-deno-crash

The important piece of the reproduction is that `i18n` is being imported from a different file in `next.config.mjs`:
```mjs
import i18n from "./i18n.mjs";
import nextTranslate from "next-translate-plugin";

/** @type {import("next").NextConfig} */
const config = {
  reactStrictMode: true,
  i18n
};

export default nextTranslate(config);
```

Moving it to be the `next.config.mjs` fixes the crash (but then the translations do not work)

Removing `nextTranslate()` wrap around `config` also fixes the crash (but then the translations do not work)

It seems that `next-translate` (or `next-translate-plugin`) is separately trying to find the i18n file.

**Comment by MonstraG** [comment]
I may have found another issue, and it might be completely separate, but if in the repro, the `next.config.mjs` is replaced with this:
```tsx
const config = {
};

export default config;
```
(So, a completely empty config) 
 
 Then, build fails with:
 ```
 PS C:\Users\monst\Downloads\next-translate-deno-crash> pnpm run build

> next-translate-deno-crash@0.1.0 build C:\Users\monst\Downloads\next-translate-deno-crash
> next build

Ignoring signal "SIGABRT" on Windows
Ignoring signal "SIGALRM" on Windows
Ignoring signal "SIGHUP" on Windows
  ▲ Next.js 14.2.15

   Creating an optimized production build ...
Ignoring signal "SIGABRT" on Windows
Ignoring signal "SIGALRM" on Windows
Ignoring signal "SIGHUP" on Windows
Failed to compile.

Error: The file or directory is not a reparse point. (os error 4390): readlink 'C:\Users\monst\Downloads\next-translate-deno-crash\node_modules\.deno\next@14.2.15\node_modules\next\dist\pages\_app.js'


> Build failed because of webpack errors
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by MonstraG** [comment]
I've tried it on 2.0.6 and the build goes trough now, so the issue was solved somewhere in (2.0.3-2.0.6]!
